homeassistant/components/isy994/entity.py

```diff
@@ -0,0 +1,96 @@
+"""Representation of ISYEntity Types."""
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.typing import Dict
+
+
+class ISYEntity(Entity):
+    """Representation of an ISY994 device."""
+
+    _attrs = {}
+    _name: str = None
+
+    def __init__(self, node) -> None:
+        """Initialize the insteon device."""
+        self._node = node
+        self._change_handler = None
+        self._control_handler = None
+
+    async def async_added_to_hass(self) -> None:
+        """Subscribe to the node change events."""
+        self._change_handler = self._node.status.subscribe("changed", self.on_update)
+
+        if hasattr(self._node, "controlEvents"):
+            self._control_handler = self._node.controlEvents.subscribe(self.on_control)
+
+    def on_update(self, event: object) -> None:
+        """Handle the update event from the ISY994 Node."""
+        self.schedule_update_ha_state()
+
+    def on_control(self, event: object) -> None:
+        """Handle a control event from the ISY994 Node."""
+        self.hass.bus.fire(
+            "isy994_control", {"entity_id": self.entity_id, "control": event}
+        )
+
+    @property
+    def unique_id(self) -> str:
+        """Get the unique identifier of the device."""
+        # pylint: disable=protected-access
```

Comment:
This is fixed in the upcoming PR?

Reply:
Never-mind its moved code.

Reply:
The pylint? Yes, goes away when VarEvents are removed in v2.